dataikuapi/dss/ml.py

```diff
@@ -811,6 +811,63 @@ def _set_range(self, min=None, max=None, nb_values=None):
             if nb_values is not None:
                 self._algo_settings[self.name]["range"]["nbValues"] = nb_values
 
+    class Range(object):
+
+        def __init__(self, numerical_hyperparameter_settings):
+            self._numerical_hyperparameter_settings = numerical_hyperparameter_settings
+            self._range_dict = self._numerical_hyperparameter_settings._algo_settings[numerical_hyperparameter_settings.name]["range"]
+
+        def __repr__(self):
+            return "Range(min={}, max={}, nb_values={})".format(self.min, self.max, self.nb_values)
```

Comment:
To be a bit more explicit
```suggestion
            return "NumericalHyperparameterSettings.Range(min={}, max={}, nb_values={})".format(self.min, self.max, self.nb_values)
```